File: Python/CS50P/mario.py
# for _ in range(3):
#     print("#")

def main():
#     print_column(3)
# 
# def print_column(height):
# #     for _ in range(height):
# #         print("#")
#     print("#\n" * height, end="")

#     print_row(4)
# 
# def print_row(width):
#     print("?"*width)

    print_square(3)

def print_square(size):
    for i in range(size):
        print_row(size)

def print_row(width):
    print("#" * width)

main()

File: Python/CS50P/C/mario.c
#include <stdio.h>

int assertInput(){
  int inInt;
  printf("Enter the size of the obstacle: ");
  scanf("%d", &inInt);
  return inInt;
}

int printRow(int width){
  while (width > 0){
    printf("*");
    width--;
  }
  return 0;
}

int printSquare(int size){
  int width_ = size;
  while (size > 0){
    printRow(width_);
    printf("\n");
    size--;
  }
  return 0;
}

int main (){
  int size = assertInput();
  printSquare(size);
  return 0;
}
